pkg/logger/logger.go

```diff
@@ -93,13 +94,13 @@ func NewNoopLogger() *ZapLogger {
 	}
 }
 
-func NewLogger(logFormat, logLevel string) (*ZapLogger, error) {
-	if logLevel == "none" {
+func NewLogger(logConfig config.LogConfig) (*ZapLogger, error) {
```

Comment:
Possibly a better alternative than
```go
func NewLogger(logFormat, logLevel, logTimestampFormat string) (*ZapLogger, error) {
```

Reply:
This logger is a publicly exported signature (e.g. `pkg/logger`). The server config is not a publicly exported signature (e.g. `internal/server/config`).

This change will break consumers of the OpenFGA library and make it impossible for anyone to construct a Logger externally as they cannot construct a `config.LogConfig` object since it is an internal package.

Reply:
Hi @jon-whit, sorry about missing that I am using an `internal` package.
I tried to take a stab at the code by changing to the options pattern like you suggested, but I am not comfortable enough to do such refactoring. As such, I have reverted to using expanded parameters. I hope this is okay and we can move this PR forward.